README.md

```diff
@@ -8,7 +8,42 @@ components.
 
 You will need at least the following installed:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+ * LLVM 15. Depending on your distribution you may need to install `llvm-devel`
+   or `llvm-extra` to get necessary extra parts such as `wasm-ld`.
+ * typescript (`tsc`) 5.2 or later. If your distribution doesn't package
+   typescript, you can install your own with `npm install
+   --prefix=/path/to/install typescript`.
+ * `yarn`
+
+
+## Build instructions
+
+First build and install the bootstrap compiler:
+
+```
+cd /path/to/repo/compiler && make clean && prefix=/path/to/install make install STAGE=0
+```
+
+Then build the rest of the system:
+
+```
+cd /path/to/repo && make clean && make
+```
+
+If you also want to build the NPM packages locally:
+
+```
+cd /path/to/repo/packages/dev
+rm -rf node_modules && yarn install && yarn build && yarn link
+cd ../react
+rm -rf node_modules && yarn install && yarn build && yarn link
```

Comment:
Shouldn't need to wipe out node_modules. And yarn link is definitely not needed for build. This sets this repo up to be linked in to another on the filesystem if you want to develop with a local package.